scripts/printing.py

```diff
@@ -0,0 +1,129 @@
+import lxml.etree as ET
+import sys
+import os
+
+# shorthand for tabulation
+def getTabs(n):
+	str=""
+	for x in range(n):
+		str+="\t"
+	return str
+
+# shorthand for printing single and list values
+def mostrar(tabs, title, value, type=None):
```

Comment:
english

Reply:
Changed to `printValue` in https://github.com/erlerobot/HRIM/pull/16/commits/5577a84c31158740903f0f3a6c44163bab83f8e5